### ISSUE TITLE
Test Drive Eve of Destruction - Blank screen playing game on M1 Mac. Only FMV works

### ISSUE THREAD

**Issue by rmervine** [xemu-project/xemu#1263]
### Title

https://xemu.app/titles/49470075/#Test-Drive-Eve-of-Destruction

### Bug Description

Game is blank on M1 Mac when trying to launch

Steps to reproduce:
1) Load dumped ISO into game
2) You will get FMV. As soon as the FMV sequence ends, you get a blank screen with music in the background. I tried pressing buttons to starting the game and it still plays music in the background

I tried a clean HDD image from xemu and experience the same issue.

### Expected Behavior

Game should be displayed instead of blank screen. I did try other games such as Halo Combat Evolved and that plays without an issue. It seems that others have tested this on Windows which is why I'm suspecting this is a Mac issue.

### xemu Version

Version:      0.7.70
Branch:       master
Commit:       080022833db0896b8af4482e726200acd51507a0
Date:         Sat Sep 10 20:00:31 UTC 2022

### System Information

CPU:          
OS Platform:  macOS
OS Version:   Version 12.6 (Build 21G115)
Manufacturer: Apple
GPU Model:    Apple M1 Pro
Driver:       4.1 Metal - 76.3
Shader:       4.10

**Comment by KBouder** [comment]
This same issue is also present on Linux (Steam Deck). SteamOS 3.4. I did get this to work on Linux about a year ago with heavy flickering on menus and during gameplay. May be considered a regression?

**Comment by diablo581** [comment]
I am experiencing this issue on Windows as well.

**Comment by diablo581** [comment]
It doesn't work correctly in the most recent build, but it works fine in v0.6.7.

I may try the versions inbetween to see when it stopped working.

**Comment by diablo581** [comment]
Sometime after v0.7.109

**Comment by PrestonLK** [comment]
It's been a long time since I last played this with XEMU, and it seems I am also now experiencing this issue. I'm on Windows 11. I haven't tried reverting to an older version of the emulator yet to see if that fixes it.

**Comment by AzurikPerathia** [comment]
The game is broken in 0.8.10

**Comment by VincxBlox** [comment]
I hadnt updated Xemu in awhile. Just did. It worked in 0.7.104. New version:broken.

**Comment by VincxBlox** [comment]
After trying multiple versions, 0.7.115 works and 0.7.116 does not. This seems to have broke it.
```
nv2a: Implement ZCLIP_MIN,MAX with gl_ClipDistance
```

**Comment by VincxBlox** [comment]
how is this still not fixed or addressed..

**Comment by mborgerson** [comment]
> how is this still not fixed or addressed..

@VincxBlox 

Please remember that this project is maintained by volunteers who contribute in their free time. A demanding tone isn’t helpful and doesn’t speed things up. Open source works best when everyone participates constructively. If you'd like to help, please try the latest version of Xemu and let us know the current state of the issue.

**Comment by diablo581** [comment]
I believe this might be fixed in the latest version.

**Comment by rmervine** [comment]
I just tried it in the latest release as of this comment - 0.8.105. And it still has the same issue. Tired 0.6.7 but crashes for me on my M1 mac. Tried 0.7.115 per another comment. Still get black screen issue. At least those are my observations on mac.

**Comment by VincxBlox** [comment]
@rmervine will try on my windows end. and linux cuz why not.

**Comment by VincxBlox** [comment]
tl;dr its fixed in windows since 8.30 (around march)
mac is still unknown what the root problem is..

i hope this helps devs..



@mborgerson 
> Please remember that this project is maintained by volunteers who contribute in their free time. A demanding tone isn’t helpful and doesn’t speed things up.

sorry was just kinda pissed off that day..

uhh so i tried it in 8.29. it doesnt work. 8.30, it does, and still does in the latest version.

games that i know were affected and now fixed are:
burnout 3
test drive eod
yeah ik not alot

7.116 had broken it with: nv2a: Implement ZCLIP_MIN,MAX with gl_ClipDistance
8.30 fixed it with: nv2a: Explicit float representation for RCC and vertex shader W range

here a video for additional context/proof
https://streamable.com/utuk6l

i thinks we could open another ticket or edit this one more clearly for MAC specifically, including more information about the problem @rmervine. perhaps debug logs? affected games, if you have access to.. specifically burnout 3 for example i noticed have the exact same problems during that release-range. you may try it if u have access to it..

**Comment by rmervine** [comment]
Thanks for the update! I can see about debug logs. I tried to see if I had any success with Parallels running windows 11. But it keeps closing down on me. I may try to see how Linux does in Parallels or even explore using my Steam Deck.

**Comment by VincxBlox** [comment]
You could perhaps try to use proton experimental on steam to run xemu if you want, I'm not sure how all this works on MAC.On Oct 4, 2025 9:01 p.m., Ron Mervine ***@***.***> wrote:rmervine left a comment (xemu-project/xemu#1263)
Thanks for the update! I can see about debug logs. I tried to see if I had any success with Parallels running windows 11. But it keeps closing down on me. I may try to see how Linux does in Parallels or even explore using my Steam Deck.

**Comment by VincxBlox** [comment]
I have an update; no logs tho I can't find them on the OS (Batocera 38, October 16 2023)

i installed batocera 38 which is technically before 0.7.116. so no reason for this. on my Intel graphics on this NUC, it does not work, black screen before I even get a fmv. like @rmervine said. it does work in WINDOWS THO, just.. extremely slowly.

![Image](https://github.com/user-attachments/assets/1e4fbd6c-2a48-41b6-953c-71e7dd527b25)

**Comment by VincxBlox** [comment]
> This same issue is also present on Linux (Steam Deck). SteamOS 3.4. I did get this to work on Linux about a year ago with heavy flickering on menus and during gameplay. May be considered a regression?

@KBouder 

i also had such on WINDOWS too awhile ago on nvidia. i have no idea when but it was awhile ago. maybe around 2022-2023
don't know if this helps
